url_handlers_using_quarkus/src/main/java/org/acme/getting/started/linux_url_handler.java

```diff
@@ -0,0 +1,30 @@
+package org.acme.getting.started;
+import java.io.*;
+/*Adding Custom URL Handler to Linux for ide:// */
+ public class linux_url_handler{
```

Comment:
why two different static void mains and not just one in which you can based on OS do different actions ?

Reply:
Yeah i did it separately for POC......we can do for all three OS in one file....
Probably we can use: `System.getProperty("os.name")`